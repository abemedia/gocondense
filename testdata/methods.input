package main

type receiver[
	A string,
	B int,
	C bool,
] struct{}

func (r *receiver[
	A,
	B,
	C,
]) Example(
	a string,
	b int,
	c bool,
) (
	string,
	int,
	bool,
) {
	return a, b, c
}
