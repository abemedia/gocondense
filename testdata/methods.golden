package main

type rec[A string, B int, C bool] struct {
}

func (r *rec[A, B, C]) Ptr(a string, b int, c bool) (string, int, bool) {
	return a, b, c
}

func (r rec[A, B, C]) Val(a string, b int, c bool) (string, int, bool) {
	return a, b, c
}
